src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+
+  &:last-of-type {
+    padding-bottom: 0;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
```

Comment:
There is an extra space after the modifier class (e.g., '&--1 :first-child'). This space can cause the selector to target any child element rather than the intended direct child. Remove the space to ensure the selector targets the correct elements (e.g., '&--1:first-child').